src/WebJobs.Script/FileTraceWriter.cs

```diff
@@ -38,9 +40,17 @@ public FileTraceWriter(string logFilePath, TraceLevel level) : base(level)
             }
             else
             {
```

Comment:
I could also move the purge code to SetNewLogFile, so we only do it when we move to a new file. That would make it less frequent.